StellarApp/Controllers/NewsView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  NewsView.swift
+//  StellarApp
+//
+//  Created by Ramu on 4/11/19.
+//  Copyright © 2019 TingxinLi. All rights reserved.
+//
+
+import UIKit
+
+class NewsView: UIView {
+
+    lazy var NewsCollectionView: UICollectionView = {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .vertical
+        let ncv = UICollectionView.init(frame: self.bounds, collectionViewLayout: layout)
+        ncv.register(NewsCollectionViewCell.self, forCellWithReuseIdentifier: "NewsCell")
+        return ncv
+        
+    }()
+    
+    
+    override init(frame: CGRect) {
+        super.init(frame: UIScreen.main.bounds)
+        commonInit()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        commonInit()
+    }
+    
+    private func commonInit() {
+    
+        setConstraints()
+    }
+    
+    func setConstraints() {
+        self.addSubview(NewsCollectionView)
+        NewsCollectionView.backgroundColor = .clear
```

Comment:
the method is `setConstraints`, set the background color in the lazy var above.

Reply:
its been move to the newsVC